src/rewrite_app_repos.sh

```diff
@@ -98,7 +114,7 @@ do
 
   if [ -n "$repo_branch" ]; then
     echo "Building assets for ${reponame} at branch ${repo_branch}"
-    time build_assets "$repo_branch" "$reponame" &
+    time PS4="__$reponame: " build_assets "$repo_branch" "$reponame" &
```

Comment:
I added this to differentiate each task output, but maybe there's a better way to do this.